lib/presentation/localizations/app_localizations.dart

```diff
@@ -631,6 +631,27 @@ class AppLocalizations {
       args: [nbFiles, quota, firstElementName],
     );
   }
+
+  String get copy_to_my_space {
```

Comment:
pls also add translated string (or even untranslated string) to `intl_fr.arb` and other localization files.